content/en/blog/_posts/2019-09-23-san-diego-contributor-summit

```diff
@@ -0,0 +1,75 @@
+organizations] all over the world. The Contributor Summits give the community three
+chances a year to get together, work on community topics, and have hallway track
+time. The upcoming San Diego summit is expected to bring over 450 attendees, and
+will contain multiple tracks with something for everyone. The focus will be around
+contributor growth and sustainability. We're going to stop here with capacity for
+future summits; we want this event to offer value to individuals and the project.
+Getting work done and learning is hard in groups larger than this but it's a priority
+of ours that you have a productive (+fun!) time.
+
+This summit is unique as we’ve taken big moves on sustaining ourselves, the
+contributor experience event team. Taking a page from the release team’s playbook,
+we have added additional core team and shadow roles making it a natural mentoring
+(watching+doing) relationship. The shadows are expected to fill another role at
+one of the three events in 2020, and core team members to take the lead.
+In preparation for this team, we’ve open sourced our [rolebooks, guidelines,
+best practices] and opened up our [meetings] and [project board]. Our team makes up
+many parts of the Kubernetes project and takes care of making sure all voices
+are represented.
+
+Are you at KubeCon + CloudNativeCon but can’t make it to the summit? Check out
+the [SIG Intro and Deep Dive sessions] under the 'maintainer track' label during
+KubeCon + CloudNativeCon to participate in Q&A and hear what’s up with each
+Special interest Group (SIG). We’ll also record all of Contributor Summit’s
+presentation sessions, take notes in discussions, and share it back with you,
+after the event is complete.
+
+We hope to see you all at Kubernetes Contributor Summit San Diego, make sure you
+head over and [register right now]! This event will sell out - here’s your warning.
+:)
+
+Check out past blogs on [persona building around our events] and the [Barcelona summit story].
+
+[Contributor Summit San Diego 2019 Event Page]: https://events.linuxfoundation.org/events/kubernetes-contributor-summit-north-america-2019/
+[QuartYard]: https://quartyardsd.com/
+[Marriott Marquis San Diego Marina]: https://www.marriott.com/hotels/travel/sandt-marriott-marquis-san-diego-marina/?scid=bb1a189a-fec3-4d19-a255-54ba596febe2
+[individuals and organizations]: https://k8s.devstats.cncf.io/d/8/company-statistics-by-repository-group?orgId=1&var-period=y&var-metric=contributions&var-repogroup_name=All&var-companies=All
+[rolebooks, guidelines, best practices]: https://github.com/kubernetes/community/tree/master/events/events-team
+[meetings]: https://docs.google.com/document/d/1oLXv5_rM4f645jlXym_Vd7AUq7x6DV-O87E6tcW1sjU/edit?usp=sharing
+[project board]: https://github.com/orgs/kubernetes/projects/21
+[SIG Intro and Deep Dive sessions]: https://events.linuxfoundation.org/events/kubecon-cloudnativecon-north-america-2019/schedule/
```

Comment:
If you want to link to the Maintainer Track Sessions, you can use this: https://kccncna19.sched.com/overview/type/Maintainer+Track+Sessions?iframe=yes

Reply:
added